src/clp_logging/handlers.py

```
@@ -206,18 +249,20 @@ def __init__(
                 raise
 
     # override
-    def setFormatter(self, fmt: Optional[logging.Formatter]) -> None:
-        _set_formatter(self, fmt)
-
-    # override
-    def emit(self, record: logging.LogRecord) -> None:
+    def write(self, msg: str) -> None:
         try:
             if self.closed:
                 raise RuntimeError("Socket already closed")
-            msg = self.format(record)
             self.sock.send(CLPEncoder.encode_message(msg.encode()))
         except Exception:
             self.sock.close()
+
```

Did you mean to swallow the exception in the except block above?